core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/ClientOverrideConfiguration.java

```diff
@@ -439,6 +451,22 @@ default Builder retryPolicy(RetryMode retryMode) {
         Builder addMetricPublisher(MetricPublisher metricPublisher);
 
         List<MetricPublisher> metricPublishers();
+
+        /**
+         * Sets the additional execution attributes collection for this client.
+         * @param executionAttributes Execution attributes map for this client.
+         * @return This object for method chaining.
+         */
+        Builder executionAttributes(Map<ExecutionAttribute<?>, Object> executionAttributes);
```

Comment:
Let's use `Map<ExecutionAttribute<?>, ?>`

Reply:
This wont work when calling putExecutionAttribute the type generic will not be able to be converted to ?. "Error:(572, 62) java: incompatible types: T cannot be converted to capture#1 of ?"